force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +331,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
+        Integer sum = 0;
         // Loop through the list of integers
             // if the number is negative skip this iteration
-        
+        for (Integer value : numbers) {
+            if (value > 0) {
+                sum += value;
+                }
+            }
```

Comment:
Watch the indentation here, should be lined up the statement that started the bracket